weekly-mission(week2)/src/apis/signUpUser.js

```diff
@@ -0,0 +1,33 @@
+// 회원가입 관련 통신 함수들 모음
+
+const isDuplicateEmail = async (email) => {
+  const response = await fetch('https://bootcamp-api.codeit.kr/api/check-email', {
```

Comment:
api url에서 `check-email`과 같은 부분을 api endpoint라고 부르는데요~ 보통 api 함수명은 endpoint를 그대로 따라가주는게 좋은 것 같아요!
```suggestion
const checkEmail = async (email) => {
  const response = await fetch('https://bootcamp-api.codeit.kr/api/check-email', {
```

Reply:
아하 ! 알겠습니당